draft-jholland-quic-multicast.md

```diff
@@ -781,7 +783,7 @@ Not permitted:
 
 Note that when a newly connected client joins a channel, the client will only be able to receive application data carried in stream frames delivered on that channel when they have received the stream data starting from offset 0 of the stream.
 
-This usually means that new streams must be started for application data carried in channel packets whenever there might be new clients that have joined since an earlier stream started.
+This usually means that new streams must be started for application data carried in channel packets whenever there might be new clients that have joined since an earlier stream started. If the server deems it convenient, it could also send preceding data for that stream over the unicast conenction to catch the client up.
```

Comment:
```suggestion
This usually means that new streams must be started for application data carried in channel packets whenever there might be new clients that have joined since an earlier stream started. If the server deems it convenient, it could also send preceding data for that stream over the unicast connection to catch the client up.
```